types/capella/block.yaml

```diff
@@ -0,0 +1,89 @@
+Capella:
+  BeaconBlockBodyCommon:
+    # An abstract object to collect the common fields between the BeaconBlockBody and the BlindedBeaconBlockBody objects
+    type: object
+    description: "The [`BeaconBlockBody`](https://github.com/ethereum/consensus-specs/blob/master/specs/capella/beacon-chain.md#beaconblockbody) object from the CL Capella spec."
+    properties:
+      randao_reveal:
+        allOf:
+          - $ref: '../primitive.yaml#/Signature'
+          - description: "The RanDAO reveal value provided by the validator."
+      eth1_data:
+        $ref: '../eth1.yaml#/Eth1Data'
+      graffiti:
+        $ref: '../primitive.yaml#/Graffiti'
+      proposer_slashings:
+        type: array
+        items:
+          $ref: '../proposer_slashing.yaml#/ProposerSlashing'
+      attester_slashings:
+        type: array
+        items:
+          $ref: '../attester_slashing.yaml#/AttesterSlashing'
+      attestations:
+        type: array
+        items:
+          $ref: '../attestation.yaml#/Attestation'
+      deposits:
+        type: array
+        items:
+          $ref: '../deposit.yaml#/Deposit'
+      voluntary_exits:
+        type: array
+        items:
+          $ref: '../voluntary_exit.yaml#/SignedVoluntaryExit'
+      sync_aggregate:
+        $ref: '../altair/sync_aggregate.yaml#/Altair/SyncAggregate'
+
+  BeaconBlockBody:
+    allOf:
+      - $ref: '#/Capella/BeaconBlockBodyCommon'
+      - type: object
```

Comment:
block body would get a `bls_to_execution_changes` added...

Reply:
sorry should have added reference https://github.com/ethereum/consensus-specs/blob/dev/specs/capella/beacon-chain.md#beaconblockbody

Reply:
Added; thanks.